creator-onboarding/curated-checklist.md

```diff
@@ -45,49 +42,57 @@ To maintain our commitment to charitable giving, we ask artists to consider dona
 Learn more about Charitable Donation [here](https://docs.artblocks.io/creator-docs/creator-onboarding/readme/charitable-donations/).
 
 ## Approve additional payee info for primary and secondary sales⛽
-Include the wallet addresses for primary and secondary sales to be distributed. Once submitted, the details will be approved by the Art Team. If you do not have additional wallet addresses that will receive funds from primary or secondary sales you must input `0x0000000000000000000000000000000000000000` in the blank fields. 
 
+Include the wallet addresses for primary and secondary sales to be distributed. Once submitted, the details will be approved by the Art Team. If you do not have additional wallet addresses that will receive funds from primary or secondary sales you must input `0x0000000000000000000000000000000000000000` in the blank fields.
 
 ## Mint #0
+
 1. **Note**: Mint #0 will be completed with the Set Price minter. Set the minter to Set Price ETH with the set price as your Dutch Auction’s base price for Mint #0
 2. **Before** minting Mint #0, ensure that your "additional payee wallet" has been set for the same configuration that you will use it for in your project release. E.g., if you are using the "additional payee wallet" field to donate to charity at the time of mint, you must set this before minting your #0. This is done to ensure that this full functionality is tested end-to-end as part of the mint #0 process, and that there are no issues with the wallet selected for the additional payee. **Please note: Art Blocks does not currently support sending primary sales payments into a multi-sig contract and there are known issues when attempting to do so. While we plan to update our minting smart contracts in the near future to resolve this, multi-sig wallets should not be used for this purpose at this time.**
-3. Once your project information has been uploaded, **please confirm in your artist DM that you're ready for mint #0**. The Art Blocks Team will look over your project shell and then give you the go-ahead to mint #0. 
+3. Once your project information has been uploaded, **please confirm in your artist DM that you're ready for mint #0**. The Art Blocks Team will look over your project shell and then give you the go-ahead to mint #0.
 4. Mint #0 must occur before your release can be scheduled.
 
-### Initiating your MinterSuite choice
+### Payout Details
+
+Be sure that you've input information in all field forms. If you do not have an additional payee for primary or secondary sales, you may put `0x0000000000000000000000000000000000000000` in the field and 0% for the payee percentage.
```

Comment:
Are there any potential problems they could run into if they don't fill out all fields? Just wondering if we want stronger language here or examples to explain what could go wrong if they miss something.

Reply:
I'm pretty sure the form prevents submission until everything is filled out, so I don't think anything can go wrong if the artist misses anything. Adding this type of messaging to the form might be a good call though!

Reply:
resolving (as this isn't a change, but moving this information from one place to another). will make a note about potentially updating the form to be more intuitive though.